lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/failed.yaml

```diff
@@ -16,6 +16,14 @@ states:
           ticfCriBeta:
             targetState: CRI_TICF_BEFORE_NO_MATCH
 
+  FAILED_COI:
+    events:
+      next:
+        targetState: COULD_NOT-CONFIRM-DETAILS
```

Comment:
Do we need _PAGE on the end here? Also, maybe use `_` instead of `-` for consistency